### PR TITLE
do streams calculation before UpToCpuSpecificOpSet() in CPU plugin

### PR DESCRIPTION
**Pull request by wangleis**
### Details:
 - *do streams calculation before UpToCpuSpecificOpSet() in CPU plugin*
 - *add WA to skip saving throughput mode information during latency mode inference and export/import test case with different performance hint

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/plugins/intel_cpu/src/plugin.cpp [hunk 1]
```diff
@@ -460,6 +460,20 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::CNNNetwork &network, const std
...
+
+    // update the props after the perf mode translated to configs
+    // TODO: Clarify the behavior of SetConfig method. Skip eng_config or not?
+    Config conf = engConfig;
+
+    conf.readProperties(config);
+    if (is_cpu_map_available()) {
+        GetPerformanceStreams(conf, nGraphFunc);
+    }
+
```
Comment (sunxiaoxia2022) [inline src/plugins/intel_cpu/src/plugin.cpp:476]: This change can cause the number of streams is different from previous version.

File: src/plugins/intel_cpu/src/plugin.cpp [hunk 2]
```diff
@@ -460,6 +460,20 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::CNNNetwork &network, const std
     DEBUG_LOG(PrintableModel(*nGraphFunc, "org_"));
 
     Transformations transformations(nGraphFunc, enableLPT, inferencePrecision, isLegacyAPI(), snippetsMode, engConfig);
```
[the remark this answers is not in the document]
Comment (wangleis) [reply src/plugins/intel_cpu/src/plugin.cpp]: Updated. This update will change the nGraph based of streams calculation, will check the performance impact again.

File: src/plugins/intel_cpu/src/plugin.cpp (hunk 1 above)
Comment (wangleis) [reply src/plugins/intel_cpu/src/plugin.cpp:511]: Thank you for reminding. This is recorded as a known issue in review.

**Comment by wangleis** [comment]
Close this PR and follow up this change in https://github.com/openvinotoolkit/openvino/pull/18911 without WA

**Pull request closed without merging**